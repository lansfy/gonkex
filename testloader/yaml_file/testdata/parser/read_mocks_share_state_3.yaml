- name: share state for two testscase with mocks declaration MUST do nothing
  method: GET
  path: /some/path1
  request: "requestbody"
  mocksParams:
    shareState: true
  mocks:
    someservice:
      strategy: nop

- name: share state for two testscase with mocks declaration MUST do nothing
  method: GET
  path: /some/path2
  request: "requestbody"
  mocksParams:
    shareState: true
  mocks:
    someservice:
      strategy: nop
