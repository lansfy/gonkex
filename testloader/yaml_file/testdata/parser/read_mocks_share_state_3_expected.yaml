- GetName: share state for two testscase with mocks declaration MUST do nothing
  GetMethod: GET
  Path: /some/path1
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_3.yaml
  GetLineNumber: 1
  FirstTestInFile: true
  ServiceMocks:
    someservice:
      strategy: nop

- GetName: share state for two testscase with mocks declaration MUST do nothing
  GetMethod: GET
  Path: /some/path2
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_3.yaml
  GetLineNumber: 11
  LastTestInFile: true
  ServiceMocks:
    someservice:
      strategy: nop
